internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/fastclient/AvroStoreClientEndToEndTest.java

```diff
@@ -416,53 +425,39 @@ public void testFastClientGetWithDifferentHTTPVariants(
         storeMetadataFetchMode);
   }
 
-  @Test(dataProvider = "FastClient-Two-Boolean-Store-Metadata-Fetch-Mode", timeOut = TIME_OUT)
-  public void testFastClientWithLongTailRetry(
-      boolean batchGet,
-      boolean useStreamingBatchGetAsDefault,
-      StoreMetadataFetchMode storeMetadataFetchMode) throws Exception {
+  @Test(dataProvider = "FastClient-One-Boolean", timeOut = TIME_OUT)
+  public void testFastClientWithLongTailRetry(boolean batchGet) throws Exception {
     ClientConfig.ClientConfigBuilder clientConfigBuilder =
         new ClientConfig.ClientConfigBuilder<>().setStoreName(storeName).setR2Client(r2Client);
 
     if (batchGet) {
       clientConfigBuilder
           // default maxAllowedKeyCntInBatchGetReq is 2. configuring it to test different cases.
           .setMaxAllowedKeyCntInBatchGetReq(recordCnt)
-          .setUseStreamingBatchGetAsDefault(useStreamingBatchGetAsDefault);
+          .setUseStreamingBatchGetAsDefault(true);
```

Comment:
Just to confirm, we will only test the true case here?

Reply:
These tests needs to be rewritten to remove flakiness, will add the case for both true/false then. For now, this should be okay.